docs/environment-variables.md

```diff
@@ -0,0 +1,68 @@
+# Environment variables
```

Comment:
Could we link to these somewhere from the main README too?

Reply:
Good idea, it will be frustrating to have to remember the url path when looking for them, when we easily set it up to click through.